tests/GithubApp.py

```diff
@@ -98,8 +98,14 @@ def testGetPublicApp(self):
         self.assertEqual(app.updated_at, datetime(2019, 12, 10, 19, 4, 12))
         self.assertEqual(app.url, "/apps/github-actions")
 
+
+class GithubAppAuth(Framework.TestCase):
+    def setUp(self):
+        self.jwtAuthMode = True
+        super().setUp()
+
     def testGetAuthenticatedApp(self):
```

Comment:
test an be removed

Reply:
let's keep this test but also assert the warning, see this test for an example:
https://github.com/PyGithub/PyGithub/blob/db616376a51789c96afe1e6039f7aecc9b08f56e/tests/Authentication.py#L54-L61